nominees/epiestim.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "EpiEstim",
+  "aliases": [
+    ""
+  ],
+  "description": "EpiEstim is a tool to estimate the time-varying instantaneous reproduction number during epidemics. This web application generates an estimate of infectious disease transmissibility throughout an outbreak. ",
+  "website": "https://github.com/mrc-ide/EpiEstim",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://cran.r-project.org/web/packages/EpiEstim/index.html"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "The quantification of transmissibility during epidemics is essential to designing and adjusting public health responses.  This tool should help epidemiologists quantify temporal changes in the transmission intensity of future epidemics by using surveillance data.",
+      "evidenceURL": "https://academic.oup.com/aje/article/178/9/1505/89262?login=false"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositories": [
+    {
+      "name": "CRAN",
+      "url": "https://cran.r-project.org/web/packages/EpiEstim/index.html"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/mrc-ide/EpiEstim/wiki"
+    }
```

Comment:
Could we add the actual GitHub repository with the R package: https://github.com/mrc-ide/EpiEstim. Also request that the GPL license file be added here.